hack/generate/samples/internal/go/memcached-with-webhooks/e2e_test_code.go

```diff
@@ -159,77 +163,110 @@ import (
+		cmd = exec.Command("kubectl", "label", "--overwrite", "ns", "--all",
+			"pod-security.kubernetes.io/audit=restricted",
+			"pod-security.kubernetes.io/enforce-version=v1.24",
+			"pod-security.kubernetes.io/warn=restricted")
+		_, err := utils.Run(cmd)
+		ExpectWithOffset(1, err).NotTo(HaveOccurred())
+
+		By("labeling enforce the namespace where the Operator and Operand(s) will run")
+		cmd = exec.Command("kubectl", "label", "--overwrite", "ns", namespace,
+			"pod-security.kubernetes.io/audit=restricted",
+			"pod-security.kubernetes.io/enforce-version=v1.24",
+			"pod-security.kubernetes.io/enforce=restricted")
+		_, err = utils.Run(cmd)
+		Expect(err).To(Not(HaveOccurred()))
+
+		By("uncommenting all sections with 'monitoring' to enable operator custom metrics")
+		err = utils.ReplaceInFile("controllers/memcached_controller.go",
+			"//"+monitoringImportFragment, monitoringImportFragment)
+		Expect(err).To(Not(HaveOccurred()))
+
+		err = utils.ReplaceInFile("controllers/memcached_controller.go",
+			"//"+incMemcachedDeploymentSizeUndesiredCountTotalFragment, incMemcachedDeploymentSizeUndesiredCountTotalFragment)
+		Expect(err).To(Not(HaveOccurred()))
+
+		err = utils.ReplaceInFile("main.go",
+			"//"+monitoringImportFragment, monitoringImportFragment)
+		Expect(err).To(Not(HaveOccurred()))
+
+		err = utils.ReplaceInFile("main.go",
+			"//"+registerMetricsFragment, registerMetricsFragment)
+		Expect(err).To(Not(HaveOccurred()))
+	})
 
-			By("uninstalling the cert-manager bundle")
-			utils.UninstallCertManager()
+	AfterAll(func() {
+		By("uninstalling the Prometheus manager bundle")
+		utils.UninstallPrometheusOperator()
 
-			By("removing manager namespace")
-			cmd := exec.Command("kubectl", "create", "ns", namespace)
-			_, _ = utils.Run(cmd)
-		})
+		By("uninstalling the cert-manager bundle")
+		utils.UninstallCertManager()
+
+		By("removing manager namespace")
+		cmd := exec.Command("kubectl", "create", "ns", namespace)
+		_, _ = utils.Run(cmd)
+	})
 
-		It("should successfully run the Memcached Operator", func() {
+	Context("Memcached Operator", func() {
+		It("should run successfully", func() {
 			var controllerPodName string
 			var err error
 			projectDir, _ := utils.GetProjectDir()
 
-			// operatorImage store the name of the imahe used in the example
-			const operatorImage = "example.com/memcached-operator:v0.0.1"
+			// operatorImage stores the name of the image used in the example
+			var operatorImage = "example.com/memcached-operator:v0.0.1"
 
 			By("building the manager(Operator) image")
-			cmd := exec.Command("make", "docker-build", "IMG=example.com/memcached-operator:v0.0.1")
+			cmd := exec.Command("make", "docker-build", fmt.Sprintf("IMG=%s", operatorImage))
+			_, err = utils.Run(cmd)
+			ExpectWithOffset(1, err).NotTo(HaveOccurred())
+
+			By("pushing the manager(Operator) image")
+			cmd = exec.Command("make", "docker-push", fmt.Sprintf("IMG=%s", operatorImage))
```

Comment:
I'm not sure we should be pushing any images to a registry during our tests. Especially because the repository/image may not exist and could result in a test failure.

Reply:
Thanks, done.

Reply:
@everettraven Should I take any action to pass DCO? The commit is signed-off.

Reply:
@assafad It looks like it is for some reason expecting the username associated with the commit to be `assaf-admi` instead of your github username `assafad`.

Did you happen to change your username on github recently? Or maybe in your local git environment? (I'm not sure if this could be what is causing this, I've never seen this particular DCO check failure before)

Reply:
Yes, that was the case. Thank you